Modules/Quotas/src/Quotas.ConsumerApi/Controllers/QuotasController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Backbone.BuildingBlocks.API;
+using Backbone.BuildingBlocks.API.Mvc;
+using Backbone.BuildingBlocks.Application.Abstractions.Exceptions;
+using Backbone.BuildingBlocks.Application.Pagination;
+using Backbone.Modules.Quotas.Application;
+using Backbone.Modules.Quotas.Application.DTOs;
+using Backbone.Modules.Quotas.Application.Identities.Queries.GetQuotasForIdentity;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using ApplicationException = Backbone.BuildingBlocks.Application.Abstractions.Exceptions.ApplicationException;
+
+namespace Backbone.Modules.Quotas.ConsumerApi.Controllers;
+
+[Route("api/v1/[controller]")]
+[Authorize("OpenIddict.Validation.AspNetCore")]
+public class QuotasController : ApiControllerBase
+{
+    private readonly ApplicationOptions _options;
+
+    public QuotasController(IMediator mediator, IOptions<ApplicationOptions> options) : base(mediator)
+    {
+        _options = options.Value;
+    }
+
+    [HttpGet("ListIndividualQuotas")]
```

Comment:
Hmm, this results in a route like `/api/v1/Quotas/ListIndividualQuotas`. This is wrong for multiple reasons:
1. It doesn't conform to REST conventions (routes should describe resources, and not actions). While this may not always be possible, in this case it is.
2. The task description says clearly states what the route should look like (except for the "api/v1" prefix, which I usually take as granted in the tasks.
3. The route says it lists individual quota, even though it lists individual quota AND tier quota.